march_gazebo/worlds/march.world

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
```

Comment:
Do we still use this?

Reply:
I think we do here:
    <arg name="world_name" value="$(find march_gazebo)/worlds/march.world"/>